files/scripts/graphdb.sh

```diff
@@ -155,6 +155,44 @@ function cloudBackup {
   log "Backup ${BACKUP_NAME} completed successfully!"
 }
 
+function createIndicesFromFiles() {
+  local node_count=$1
+  local indicesConfigsLocation=$2
+  local success=true
+  local response
+  local repository
+
+  set -v
```

Comment:
Let's not enable the verbose output for consistency with the rest of the methods.